src/main/java/net/redstoneore/legacyfactions/Relation.java

```diff
@@ -57,7 +57,7 @@ public String toNiceName() {
 	
 	/**
 	 * Convert a string to a Relation object, defaults to NEUTRAL if unknown
-	 * @param string
+	 * @param str Relation string
```

Comment:
👍 better documentation is one of our core goals! more docs are better than none! 💃